### PR TITLE
challenges: update grasping challenge field to mongodb changes

### PR DESCRIPTION
**Pull request by TarikViehmann**
This PR updates the game report for the fixed challenge track field to fit the new format.

### CODE REVIEW

File: cfg/challenges/grasping/grasping_game.yaml [hunk 1]
```diff
@@ -20,6 +20,6 @@ llsfrb:
       phase: PRODUCTION
     # load data from latest game report with a given name
     # leave empty to always load from latest stored report
-    load-from-report: "GraspingChallenge"
+    load-from-report: "GraspingChallengeNew"
```
Comment (snoato) [inline cfg/challenges/grasping/grasping_game.yaml:23]: Are we giving a new name for backwards compatibility (which should be broken anyhow)?

Comment (TarikViehmann) [reply cfg/challenges/grasping/grasping_game.yaml:23]: i thought that this would avoid confusion also regarding existing databases.

Comment (snoato) [reply cfg/challenges/grasping/grasping_game.yaml:23]: Fair point!

**Review by snoato — CHANGES_REQUESTED** [review]
Thanks!

**Review by snoato — APPROVED** [review]

**Pull request merged**